common/http/platform/beast/http.cpp

```diff
@@ -169,6 +169,15 @@ error::Error Client::AsyncCall(
 	return error::NoError;
 }
 
+void Client::CallHandler(ResponseHandler handler) {
+	// This function exists to make sure we have a copy of the handler we're calling. This is
```

Comment:
I'd emphasize that the argument is actually the copy (if I understand it correctly) because it's not entirely obvious.

Reply:
Good point, fixed!